### PR TITLE
Timestamp output

### PR DESCRIPTION
**Pull request by ghindle**
NV/chrome-devtools-autosave-server#6

### CODE REVIEW

**Comment by NV** [comment]
toLocaleString is too verbose:

``` javascript
new Date().toLocaleString() // "Fri Jan 13 2012 17:13:09 GMT+0400 (MSK)"
```

Would be better to have what you originally suggested: `2012-01-13 17:13:09`

**Comment by ghindle** [comment]
Try that, I've amended the commit as opposed to making a new one.

File: index.js [hunk 1]
```diff
@@ -79,7 +79,13 @@ function start(routes, port, address) {
             stream.on('close', function() {
                 response.writeHead(200);
                 response.end('OK\n');
-                console.log('Saved a ' + request.headers['x-type'] + ' to ' + path);
+
+                var dateObj = new Date(),
+                    date, time;
+                date = ('0' + dateObj.getDate()).slice(-2) + '-' + ('0' + (dateObj.getMonth()+1)).slice(-2) + '-' + dateObj.getFullYear();
+                time = ('0' + dateObj.getHours()).slice(-2) + ':' + ('0' + dateObj.getMinutes()).slice(-2) + ':' + ('0' + dateObj.getSeconds()).slice(-2);
```
Comment (NV) [inline index.js]: dateObj.toLocaleTimeString() seems to do the same.

File: index.js [hunk 2]
```diff
@@ -79,7 +79,13 @@ function start(routes, port, address) {
             stream.on('close', function() {
                 response.writeHead(200);
                 response.end('OK\n');
-                console.log('Saved a ' + request.headers['x-type'] + ' to ' + path);
+
+                var dateObj = new Date(),
+                    date, time;
+                date = ('0' + dateObj.getDate()).slice(-2) + '-' + ('0' + (dateObj.getMonth()+1)).slice(-2) + '-' + dateObj.getFullYear();
```
Comment (NV) [inline index.js:85]: I like zero padding.

File: index.js [hunk 3]
```diff
@@ -79,7 +79,11 @@ function start(routes, port, address) {
...
+                console.log(date + ' ' + dateObj.toLocaleTimeString() + ': Saved a ' + request.headers['x-type'] + ' to ' + path);
```
Comment (ghindle) [inline index.js:86]: Good point! Didn't think about that. I've amended it now.

**Comment by NV** [comment]
Merged, thanks!

**Comment by NV** [comment]
I’ve found the timestamps noisy and removed them. You can prepend it yourself: http://stackoverflow.com/questions/21564/is-there-a-unix-utility-to-prepend-timestamps-to-lines-of-text

**Comment by NV** [comment]
Even better:

```
➤ node index.js | node timestamps.js 
18-03-2012 00:36:47: DevTools Autosave 1.0.0 is running on http://127.0.0.1:9104
```

timestamps.js:

``` javascript
process.stdin.resume();
process.stdin.setEncoding('utf8');
process.stdin.on('data', function(chunk) {
    var dateObj = new Date(),
        date, time;
    date = ('0' + dateObj.getDate()).slice(-2) + '-' + ('0' + (dateObj.getMonth()+1)).slice(-2) + '-' + dateObj.getFullYear();
    process.stdout.write(date + ' ' + dateObj.toLocaleTimeString() + ': ' + chunk);
});
```

Pipes are awesome!

**Pull request merged**